README.md

```diff
@@ -14,6 +14,7 @@ An easier way to use PHPUnit with [CodeIgniter](https://github.com/bcit-ci/CodeI
 * Well documented.
 
 ![Screenshot: Running tests on NetBeans 8.1](https://pbs.twimg.com/media/CUUmhxWVAAAwx3b.png)
+![Screenshot: Test result generated with phing](https://github.com/biechao/ci-phpunit-test/blob/master/docs/test_result.png?raw=true)
```

Comment:
@biechao The screenshot path is your repository. Fix it, please.

Reply:
Where should suppose to place my screenshot?

Reply:
@biechao How about like this?
```
![Screenshot: Test result generated with phing](docs/imgs/phoing-test-result.png)
```

See https://raw.githubusercontent.com/kenjis/ci-phpunit-test/master/README.md